### PR TITLE
[CUBRIDMAN-111] Add detailed description of cubrid broker status

### PR DESCRIPTION
**Pull request by airnet73**
http://jira.cubrid.org/browse/CUBRIDMAN-111

Purpose
You can check broker status information with cubrid broker status.
Also, options are provided in the cubrid broker status, and various information can be checked depending on the options.
However, additional explanation is needed as there is a lack of detailed explanation and omission of status information of the BROKER.

Added **SQL_LOG_MODE** description in the result of "**cubrid broker status -b -t -s 1**"
Added **ACCESS_MODE** and **SQL_LOG** descriptions in the result of "**cubrid broker status -b -f -l 2**"

Implementation
N/A

Remarks
N/A

### CODE REVIEW

File: en/admin/control.rst [hunk 1]
```diff
@@ -1131,6 +1131,8 @@ Meaning of added columns in code above is as follows:
 *   AS(Ns-W): The number of CAS that the client belongs to has been waited for N seconds.
 *   AS(Ns-B): The number of CAS that the client belongs to has been Busy for N seconds.
 *   CANCELED: The number of queries have cancelled by user interruption since the broker starts (if it is used with the **-l** *N* option, it specifies the number of accumulations for *N* seconds).
+*   ACCESS_MODE: The mode in which the broker connects to the DB. Broker mode is divided into three types: Read Write, Read Only, and Standby Only. For details, see :ref:`broker-mode`.
```
Comment (kisoo-han) [inline en/admin/control.rst]: Broker mode is classified into three types: Read Write, Read Only, and Standby Only. For details, see :ref:`broker-mode`.

File: en/admin/control.rst [hunk 2]
```diff
@@ -1131,6 +1131,8 @@ Meaning of added columns in code above is as follows:
...
+*   SQL_LOG: SQL logging mode of CAS. SQL LOG is divided into five types: ALL, OFF, ERROR, NOTICE, and TIMEOUT. For details, see :ref:`SQL_LOG <sql-log>` .
```
Comment (kisoo-han) [inline en/admin/control.rst]: I prefer **classified** rather than **divided**

**Review by kisoo-han — APPROVED** [review]

File: en/admin/control.rst (hunk 1 above)
Comment (airnet73) [reply en/admin/control.rst]: Thank you for comment.

File: en/admin/control.rst (hunk 2 above)
Comment (airnet73) [reply en/admin/control.rst]: Thank you for comment.

**Pull request merged**